src/test/java/org/xwiki/git/script/GitScriptServiceTest.java

```diff
@@ -81,37 +79,37 @@ public void setupRepository() throws Exception
     public void getRepositoryAndFindAuthors() throws Exception
     {
         GitScriptService service = this.componentManager.getInstance(ScriptService.class, "git");
-        Repository repository = service.getRepository(this.testRepository.getAbsolutePath(), TEST_REPO_CLONED,
-                "test author", "TestAccessCode");
+        Repository repository = service.getRepository(this.testRepository.getAbsolutePath(), TEST_REPO_CLONED);
         assertEquals(true, new Git(repository).pull().call().isSuccessful());
-
-        CommitFinder finder = new CommitFinder(repository);
-        CommitCountFilter count = new CommitCountFilter();
-        finder.setFilter(count);
-        finder.find();
-
-        assertEquals(1, count.getCount());
-
+        // Now find authors
         Set<PersonIdent> authors = service.findAuthors(repository);
         assertEquals(1, authors.size());
         assertEquals("test author", authors.iterator().next().getName());
     }
 
     @Test
-    public void getCountCommits() throws Exception
+    public void getRepositoryAndCountCommits() throws Exception
     {
         GitScriptService service = this.componentManager.getInstance(ScriptService.class, "git");
         Repository repository = service.getRepository(this.testRepository.getAbsolutePath(), TEST_REPO_CLONED,
-                "test author", "TestAccessCode");
+            "test author", "TestAccessCode");
```

Comment:
why use credentials for a `getRepositoryAndCountCommits`  test?

Reply:
You need a test to test credentials.

Reply:
I have removed credentials from all of `GitScriptServiceTest` since the credentials are tested by `DefaultGitManagerTest`. Let me know if this is not good.